### PR TITLE
add cpu hour in project

### PR DESCRIPTION
**Pull request by FanchTheSystem**

### CODE REVIEW

**Comment by FanchTheSystem** [comment]
My own test are ok, this new field in project will be used by billing https://github.com/genouest/genouestaccountmanager/pull/265

@osallou please review

File: manager2/src/app/admin/projects/projects.component.html [hunk 1]
```diff
@@ -35,8 +35,12 @@ <h3>Project creation</h3>
...
         </div>
         <div class="col-sm-2">
-          <label for="project_size">Size (GB)</label>
-          <input placeholder="project size in G" type="number" id="project_size" [value]="default_size" [ngModelOptions]="{standalone: true}" [(ngModel)]="new_project.size" class="form-control"/>
+            <label for="project_size">Size (GB)</label>
+            <input placeholder="project size in GB" type="number" id="project_size" [value]="default_size" [ngModelOptions]="{standalone: true}" [(ngModel)]="new_project.size" class="form-control"/>
+        </div>
+        <div class="col-sm-2">
+            <label for="project_cpu">Cpu (Hour)</label>
+            <input placeholder="project cpu in G" type="number" id="project_cpu" [value]="default_cpu" [ngModelOptions]="{standalone: true}" [(ngModel)]="new_project.cpu" class="form-control"/>
```
Comment (osallou) [inline manager2/src/app/admin/projects/projects.component.html:43]: In G?

File: manager2/src/app/project/project.component.html [hunk 2]
```diff
@@ -18,9 +18,15 @@ <h3>Request a new project creation</h3>
...
-            <input placeholder="project size in G" type="number" id="new_project_size" [value]="default_size" [ngModelOptions]="{standalone: true}" [(ngModel)]="new_project.size" class="form-control"/>
-            <small id="nameHelp" class="form-text text-muted"><i>Optional, for information only</i></small>
+              <label for="new_project_size">Size (GB)</label>
+              <input placeholder="project size in GB" type="number" id="new_project_size" [value]="default_size" [ngModelOptions]="{standalone: true}" [(ngModel)]="new_project.size" class="form-control"/>
+              <small id="nameHelp" class="form-text text-muted"><i>Optional, for information only</i></small>
+          </div>
+
+          <div class="form-group row col-sm-8">
+              <label for="new_project_cpu">Cpu (Hour)</label>
+              <input placeholder="project cpu in G" type="number" id="new_project_cpu" [value]="default_cpu" [ngModelOptions]="{standalone: true}" [(ngModel)]="new_project.cpu" class="form-control"/>
```
Comment (osallou) [inline manager2/src/app/project/project.component.html:28]: Cpu in g again

File: manager2/src/app/project/project.component.html [hunk 3]
```diff
@@ -163,10 +171,16 @@ <h3>Project overview</h3>
...
-              </td>
-              <td>{{selectedProject.size}} Gb</td>
+                <td><b>Size</b>&nbsp;
+                    <span class="glyphicon glyphicon-question-sign" title="Storage quota" tooltip></span>
+                </td>
+                <td>{{selectedProject.size}} GB</td>
+            </tr>
+            <tr>
+                <td><b>Cpu</b>&nbsp;
+                    <span class="glyphicon glyphicon-question-sign" title="Storage quota" tooltip></span>
```
Comment (osallou) [inline manager2/src/app/project/project.component.html:181]: Not storage quota here

**Review by osallou — CHANGES_REQUESTED** [review]

File: manager2/src/app/admin/projects/projects.component.html (hunk 1 above)
Comment (FanchTheSystem) [reply manager2/src/app/admin/projects/projects.component.html]: fixed: 5ecb403008f78cc3716119dfae809234dba93363

File: manager2/src/app/project/project.component.html (hunk 2 above)
Comment (FanchTheSystem) [reply manager2/src/app/project/project.component.html]: fixed: 5ecb403008f78cc3716119dfae809234dba93363

File: manager2/src/app/project/project.component.html (hunk 3 above)
Comment (FanchTheSystem) [reply manager2/src/app/project/project.component.html]: fixed: 5ecb403008f78cc3716119dfae809234dba93363

**Pull request merged**